### DISCUSSION TITLE
RTX 3080 is not compatible with pytorch 1.7.1 which is fixed for bindsnet

### DISCUSSION THREAD

**Discussion by kor333** [BindsNET/bindsnet#470] [Q&A]
With the following messages.

/home/ubuntu/miniconda3/envs/binds/lib/python3.6/site-packages/torch/cuda/init.py:104: UserWarning:
GeForce RTX 3080 with CUDA capability sm_86 is not compatible with the current PyTorch installation.
The current PyTorch install supports CUDA capabilities sm_37 sm_50 sm_60 sm_70 sm_75.
If you want to use the GeForce RTX 3080 GPU with PyTorch, please check the instructions at https://pytorch.org/get-started/locally/

Could you help me with this problem?

**Comment by Hananel-Hazan** [comment]
Thank you for checking BindsNET on the third NVIDIA series. 

This is seems a PyTorch issue, can you check if PyTorch 1.8 works well on your card? if it does, downgrade to 1.7.1 and run the same thing, does it work?
